packages/Python/lldbsuite/test/lang/swift/class_error/main.swift

```diff
@@ -0,0 +1,31 @@
+// Let's try to make sure that frame var prints the dynamic type of a class
+// (and a subclass) conforming to Error correctly (and that we don't crash).
+// <rdar://problem/41025993>
+
+class MyErr : Error {
+  var x : Int
+
+  init(_ x : Int) {
+    self.x = x
+  }
+}
+
+class MyOtherErr : MyErr {}
+
+func f<T>(_ Pat : T) -> T {
+  return Pat //%self.expect('frame variable -d run -- Pat', substrs=['MyErr', 'x = 23'])
+}
+
+func g<T>(_ Pat : T) -> T {
+  return Pat //%self.expect('frame variable -d run -- Pat', substrs=['MyOtherErr', 'x = 42'])
+}
+
+func main() -> Int {
+  let foo : MyErr = MyErr(23)
+  let goo : MyErr = MyOtherErr(42)
+  let patatino = f(foo)
+  let tinky = g(goo)
```

Comment:
It's possible that this won't work because of an unrelated bug, but
```
func h<U, V>(_ tuple : (U, V)) -> (U, V) {
  return tuple //%self.expect('frame variable -d run -- Pat', substrs=['MyErr', 'x = 23', 'MyOtherErr', 'x = 42'])
}

h((foo, goo))
```
would be an interesting testcase, too.

Reply:
This is failing with 

```
(lldb) frame var tuple
(U, V) tuple = <Unable to determine byte size.>
```
which I think is unrelated to my patch (it actually fails before getting to the resolution).